src/splitobs.jl

```diff
@@ -40,26 +40,31 @@ end
 """
     splitobs(data; at, shuffle=false) -> Tuple
 
-Split the `data` into multiple subsets proportional to the
-value(s) of `at`. 
+Partition the `data` into two or more subsets.
+When `at` is a number (between 0 and 1) this specifies the proportion in the first subset.
+When `at::Tuple`, each entry specifies the proportion an a subset, with the last having `1-sum(ast)`.
```

Comment:
`sum(ast)` is a typo
```suggestion
When `at` is a tuple, each entry specifies the proportion an a subset, with the last having `1-sum(at)`.
```